### ISSUE TITLE
Paper's outline

### ISSUE THREAD

**Issue by mdipirro** [RogueOneQF/quiz-fight-paper#1]
Possible paper's outline as follows:

1. Introduction 
2. Design --> something more detailed about the application
3. Server --> server's design choices
4. Client --> client's design choices
    - Firebase Cloud Messaging
    - Google Play Games
    - Facebook
    - Connectivity (EDIT)
5. Limitations (or further work, intended as a way to show how QuizFight could be improved)

**Comment by EmanueleC** [comment]
If you want I can merge (*and check*) the branches already present by following this structure.

**Comment by mdipirro** [comment]
Yes, please. Probably there will be some conflicts in main.tex due to the lack of a structure when I started writing.

**Comment by EmanueleC** [comment]
I have created a new branch: https://github.com/RogueOneQF/quiz-fight-paper/tree/outline for this issue.

**Comment by EmanueleC** [comment]
Now it seems fine. I have added a very small part on facebook login.

**Comment by mdipirro** [comment]
Ok. If it's fine delete the other branches and create a pr for this issue. I'll merge that, if the Facebook part is good.
